backend/coins/coin/conversions.go

```diff
@@ -8,10 +8,18 @@ import (
 	"github.com/digitalbitbox/bitbox-wallet-app/backend/rates"
 )
 
+// Btc2Sat is the sat equivalent of 1 BTC.
+const Btc2Sat = 1e8
+
 // FormatAsPlainCurrency handles formatting for currencies in a simplified way.
 // This should be used when `FormatAsCurrency` can't be used because a simpler formatting is needed (e.g. to populate forms in the frontend).
-func FormatAsPlainCurrency(amount *big.Rat, coinUnit string) string {
+func FormatAsPlainCurrency(amount *big.Rat, coinUnit string, btcUnit string) string {
 	var formatted string
+	if btcUnit == UnitSats || btcUnit == "t"+UnitSats {
```

Comment:
question: does `"t"+UnitSats` mean it could be `tsat`?

Reply:
Yep. When using testnet BTC was already formatted as TBTC. I did the same with sat -> tsat